internal/pkg/describe/service.go

```diff
@@ -296,6 +300,32 @@ func (d *ecsServiceDescriber) ServiceConnectDNSNames() ([]string, error) {
 	return service.ServiceConnectAliases(), nil
 }
 
+// DeploymentConfigAlarmNames returns the rollback alarm names of a service.
+func (d *ecsServiceDescriber) DeploymentConfigAlarmNames() ([]string, error) {
```

Comment:
what about 
```suggestion
func (d *ecsServiceDescriber) RollbackAlarmNames() ([]string, error) {
```

Reply:
changed the name but left it as public for now, so we can reexamine all of the `svcDescr` methods together because they're so similar.